### PR TITLE
Add conditional for alternate donut title

### PR DESCRIPTION
**Pull request by zlayne**
Signed-off-by: Zack Layne <zlayne@redhat.com>

Add conditional for alternate default donut text as seen below:
Screen Shot 2021-06-02 at 9 40 10 AM

### CODE REVIEW

File: src/AcmCharts/AcmDonutChart/AcmDonutChart.stories.tsx [hunk 1]
```diff
@@ -47,6 +53,16 @@ export const DonutChart = () => {
             />
             <AcmDonutChart title="Pods" description="Overview of pod count and status" data={podData} />
             <AcmDonutChart title="Cluster status" description="Overview of cluster status" data={clusterData} />
+            <AcmDonutChart
+                title="Cluster issues"
+                description="Overview of cluster issues"
+                data={insightData}
+                donutTitle={{
```
Comment (jlpadilla) [inline src/AcmCharts/AcmDonutChart/AcmDonutChart.stories.tsx:60]: Minor suggestion: How about using `donutLabel` instead to make it clear that it's different from the card title?

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

Added a minor suggestion, optional.

**Comment by zlayne** [comment]
/unhold
`/unhold`

**Pull request merged**